### PR TITLE
homework is done (cal + wordSwapZ)

### PR DESCRIPTION
**Pull request by hastos30**

### CODE REVIEW

File: src/Lesson4/Main.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package Lesson4;
+
+import java.util.Scanner;
+
+public class Main {
+    static Scanner scanner = new Scanner(System.in);
```
Comment (shaitanaleksander) [inline src/Lesson4/Main.java:6]: bad practice, is scanner will be closed, app will fail

File: src/Lesson4/Main.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            if (Character.isUpperCase(c)) {
+                textChar[i] = Character.toLowerCase(c);
+            } else if (Character.isLowerCase(c)) {
+                textChar[i] = Character.toUpperCase(c);
+            }
+        }
+        String newText = new String(textChar);
+        newText = newText.replace("z", "*");
+
+        System.out.println(newText);
```
Comment (shaitanaleksander) [inline src/Lesson4/Main.java:29]: *ЮУКЯд*ІБОТ*,ЕЖОБ*ОЩ*Я*ЕН*!ЬЛАКСОМ
looks like the wrong answer

File: src/Lesson4/Main.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            } else if (operation == '-') {
+                System.out.println(num1 + " - " + num2 + " = " + (num1 - num2));
+            } else if (operation == '*') {
+                System.out.println(num1 + " * " + num2 + " = " + (num1 * num2));
+            } else if (operation == '/') {
+                System.out.println(num1 + " / " + num2 + " = " + (num1 / num2));
+            } else {
+                System.out.println("Ошибка ввода! Повторите операцию!");
+                calculator();
+            }
```
Comment (shaitanaleksander) [inline src/Lesson4/Main.java:50]: why not use a switch operator?

File: src/Lesson4/Main.java (hunk 1 above)
Comment (hastos30) [reply src/Lesson4/Main.java:6]: app works - I tested it

File: src/Lesson4/Main.java (hunk 2 above)
Comment (hastos30) [reply src/Lesson4/Main.java:29]: the wordSwapZ() method outputs : *ЮУКЯд*ІБОТ*,ЕЖОБ*ОЩ*Я*ЕН*!ЬЛАКСОМ

File: src/Lesson4/Main.java (hunk 3 above)
Comment (hastos30) [reply src/Lesson4/Main.java:50]: It is more convenient and familiar to me to use "if"

File: src/Lesson4/Main.java (hunk 2 above)
Comment (hastos30) [reply src/Lesson4/Main.java:29]: > _ЮУКЯд_ІБОТ*,ЕЖОБ_ОЩ_Я_ЕН_!ЬЛАКСОМ looks like the wrong answer

the answer is correct, but not all characters are displayed in the comments

File: src/Lesson4/Main.java (hunk 3 above)
Comment (hastos30) [reply src/Lesson4/Main.java:50]: > why not use a switch operator?

the answer is correct, but not all characters are displayed in the comments

File: src/Lesson4/Main.java (hunk 1 above)
Comment (hastos30) [reply src/Lesson4/Main.java:6]: > bad practice, is scanner will be closed, app will fail

the answer is correct, but not all characters are displayed in the comments

File: src/Lesson4/Main.java (hunk 2 above)
Comment (hastos30) [reply src/Lesson4/Main.java:29]: *ЮУКЯд*ІБОТ*,ЕЖОБ*ОЩ*Я*ЕН*!ЬЛАКСОМ

File: src/Lesson4/Main.java (hunk 3 above)
Comment (hastos30) [reply src/Lesson4/Main.java:50]: > why not use a switch operator?

because I'm used to using the "if" statement

**Pull request closed without merging**